lib/config.js

```diff
@@ -1,13 +1,13 @@
 const envVars = process.env;
 module.exports = {
   NODE_ENV: envVars.NODE_ENV,
-  KMT_SECRET: envVars.KMT_SECRET,
-  KAT_ERROR_PATH: envVars.KAT_ERROR_PATH || '/error',
-  IGNORE_AUTH_TOKEN: envVars.IGNORE_AUTH_TOKEN || false,
-  HOST: envVars.HOST,
-  LOGIN_URL: envVars.LOGIN_URL,
-  LOGOUT_URL: envVars.LOGOUT_URL,
-  BASE_PATH_URL: envVars.BASE_PATH_URL,
+  KMT_SECRET: envVars.KAT_KMT_SECRET || envVars.KMT_SECRET,
+  KAT_ERROR_PATH: envVars.KAT_KAT_ERROR_PATH || envVars.KAT_ERROR_PATH || '/error',
```

Comment:
Well my merge was not that smart, mind if we leave in both till we've moved over and then delete?